### PR TITLE
Add documentation for KNX IP secure

### PR DESCRIPTION
**Pull request by marvin-w**
## Proposed change

Adds documentation on how to configure IP Secure and how to obtain the passwords and keys.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/68906
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/knx.markdown [hunk 1]
```diff
@@ -121,6 +115,35 @@ Connection parameters are set up when adding the integration and can be changed
...
+
+You can configure the IP Secure credentials either manually or by providing a `.knxkeys` file, which you can obtain by exporting the keyring in ETS as seen in the screenshot below.
+
+<p class='img'>
+  <img src='/images/integrations/knx/export_keyring_ets.png' alt='Screenshot: Export Keyring in ETS5'>
+</p>
+
+The `.knxkeys` file has to be placed in `config/.storage/knx/yourfile.knxkeys`.
+
+If you decide to configure IP Secure manually you will need the user ID (starts with 2), the user password and the device authentication password.
```
Comment (spacegaier) [inline source/_integrations/knx.markdown]: "starts with 2" = does that mean the digit "2" followed by some more digits or characters or is the first user ID the number 2, the next one would be 3?

Comment (marvin-w) [reply source/_integrations/knx.markdown]: Hehe, was about to adjust this :). You can define 2 and 2 is the default. It depends on the tunneling channel that you would like to use though.

File: source/_integrations/knx.markdown [hunk 2]
```diff
@@ -121,6 +115,37 @@ Connection parameters are set up when adding the integration and can be changed
...
+The KNX integration currently supports IP secure tunneling.
+IP secure via routing and data secure are currently not supported.
+
+In order to use IP Secure you will have to chose "Tunneling" -> "TCP with IP Secure" in the config flow.
+
+You can configure the IP Secure credentials either manually or by providing a `.knxkeys` file, which you can obtain by exporting the keyring in ETS as seen in the screenshot below.
+
+<p class='img'>
+  <img src='/images/integrations/knx/export_keyring_ets.png' alt='Screenshot: Export Keyring in ETS5'>
+</p>
```
Comment (frenck) [inline source/_integrations/knx.markdown]: Please use markdown as much as possible

File: source/_integrations/knx.markdown [hunk 3]
```diff
@@ -121,6 +115,37 @@ Connection parameters are set up when adding the integration and can be changed
...
+
+If you decide to configure IP Secure manually you will need the user ID, the user password and the device authentication password.
+
+The user id 0 is reserved and the user id 1 is used for management tasks, thus you will need to specify a user id that is 2 or higher according to the tunneling channel you would like to use. 
+
+The following screenshot will show how you can get the device authentication password in ETS.
+
+<p class='img'>
+  <img src='/images/integrations/knx/device_authentication_password.png' alt='Screenshot: Obtain device authentication password in ETS'>
+</p>
```
Comment (frenck) [inline source/_integrations/knx.markdown]: Same as above

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/knx.markdown (hunk 2 above)
Comment (marvin-w) [reply source/_integrations/knx.markdown]: Looked at a wrong example, fixed now

**Review by frenck — APPROVED** [review]
Thanks, @marvin-w 👍

**Pull request merged**